noxfile.py

```diff
@@ -75,7 +88,27 @@ def test(session):
     if _should_install(session):
         session.install("-e", ".[test]")
     _compile_translations(session)
-    session.run("pytest", *session.posargs)
+    session.run("pytest", "-k", "not a11y", *session.posargs)
+
+
+def a11y(session):
+    """Run the accessibility test suite."""
+    if _should_install(session):
+        session.install("-e", ".[test]")
+        # Install the drivers that Playwright needs to control the browsers
+        session.run("playwright", "install")
```

Comment:
re: use in ci/gitpod. it is nice to have one command that works in both.  i've found it easiest to install playwright `--with-deps` which combines `install` and `install-deps` when `CI` or `GITPOD_WORKSPACE_ID` environment variables are found.

```python
PW = "playwright", "install"
if os.environ.get("CI") OR os.environ.get("GITPOD_WORKSPACE_ID"):
  PW += "--with-deps",
session.run(*PW)
```

you can test it work with `CI=1 nox -s test`